README.md

```diff
@@ -54,7 +54,6 @@ You can also include it in a `requirements.yml` file and install it with `ansibl
 ---
 collections:
   - name: amazon.aws
-    version: 0.1.1
```

Comment:
Looks like I missed syncing these changes from community, what do you think of this section?
https://github.com/ansible-collections/community.aws/blame/master/README.md#L208:L224

Reply:
It's up to you if you want that level of detail. I was just taking out  the versions so that nobody felt  the need to update them to match the current collection version, and because it wasn't required. The detail is in docs.ansible.com, but up to you if you also want it here.